benchopt/tests/test_benchmark_features.py

```diff
@@ -351,16 +350,20 @@ def get_result(self, **data):
 
 @pytest.mark.parametrize("test_case", ["without_data_home", "with_data_home"])
 def test_paths_config_key(test_case):
+    data_path = os.path.normpath("/path/to/data")
+    home_data_path = os.path.normpath("/path/to/home_data")
+    relative_data_path = os.path.normpath("path/to/data")
```

Comment:
This is not the right test case as the idea with `relative_data_path` is that it should not be absolute (start with `/`).
Putting `normpath` will make it an absolute path.

Maybe keep the original string but compare them with conversion to `Path` objects.